media/repository.html.ejs

```diff
@@ -13,86 +13,93 @@
+            </div>
+            <div class="extraItem">
+              <p class="extraItemLabel">Version</p>
+              <vscode-text-field
+                class="extraItemInput"
+                stateName="version"
+                placeholder="1.0.0"
+                >
+            </div>
+            <div class="extraItem onlyForPilet hide">
+              <p class="extraItemLabel">Piral Package</p>
+              <vscode-text-field
+                class="extraItemInput"
+                stateName="piralPackage"
+                placeholder="sample-piral"
+                >
+            </div>
+            <div class="extraItem onlyForPilet hide">
+              <p class="extraItemLabel">NPM Registry</p> 
+              <vscode-text-field
+                class="extraItemInput"
+                stateName="npmRegistry"
+                placeholder="https://registry.npmjs.org/"
+                >
+            </div>
+          </div>  
+          <div class="bundlers">
+            <p class="columnTitle">Select bundler <span class="errorBundler errorMessage hide">[required]</span></p>
+            <% bundlers.forEach((bundler, index) => { %>
+              <div
+                key="bundler_<%= index %>"
+                class="card bundler"
+                bundler="<%= bundler.type %>"
+                >
+                <img class="selectedCardTag" src="<%= images.selectedItemIcon %>" />
+                <div class="cardTitle">
+                  <img class="cardTitleIcon" src="<%= bundler.icon %>" />
+                  <p class="cardTitleTxt"><%= bundler.title %></p>
+                </div>
+                <div class="cardDescription"><%= bundler.description %></div>
+              </div>
+            <% }); %>
           </div>
         </div>
-        <div class="cards containerColumn">
-          <p class="columnTitle">Select bundler <span class="errorBundler errorMessage hide">[required]</span></p>
-          <% bundlers.forEach((bundler, index) => { %>
-            <div
-              key="bundler_<%= index %>"
-              class="card bundler"
-              bundler="<%= bundler.type %>"
-              >
-              <img class="selectedCardTag" src="<%= images.selectedItemIcon %>" />
-              <div class="cardTitle">
-                <img class="cardTitleIcon" src="<%= bundler.icon %>" />
-                <p class="cardTitleTxt"><%= bundler.title %></p>
-              </div>
-              <div class="cardDescription"><%= bundler.description %></div>
-            </div>
-          <% }); %>
         </div>
       </div>
-      <a class="btn create" title="Start Scaffolding">Scaffold Project</a>
-    </div>
+      <div class="btn-container">
+        <vscode-button class="btn create" title="Start Scaffolding" href="#">Scaffold Project</vscode-button>
+      </div>
   </div>
   <% scripts.forEach((script) => { %>
-     <script src="<%= script %>"></script>
+     <script type="module" src="<%= script %>"></script>
```

Comment:
Are we sure that every script is a module? If so - cool - if not, could we just instead of having an `Array<string>` have `Array<{ url: string; type: 'module' | 'application/javascript' }>`?